ui_components/src/home/Home.css

```diff
@@ -800,9 +800,23 @@ body {
     margin-left: 5px;
     margin-right: 5px;
 }
-.news_item_details {
+.news_item_edit {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 12px;
+    font-weight: bold;
+    color: #6a5e46;
```

Comment:
TODO at some point - extract these colors to vars so we could more easily re-theme the logged out view if desired

Reply:
Marking as to-do.